esbuild-plugin/src/esbuild-plugin.js

```diff
@@ -1,10 +1,12 @@
 import { parse } from "path";
 import { readFile } from "fs/promises";
-import * as lightning from "lightningcss";
 import { createGenerator } from "@unocss/core";
 import { classes } from "@warp-ds/css/component-classes/classes";
 import { presetWarp } from "@warp-ds/uno";
+import browserslist from "browserslist";
+import * as lightning from "lightningcss";
 
+const targets = lightning.browserslistToTargets(browserslist("supports es6-module and > 0.25% in NO and not dead"))
```

Comment:
The browserslist config ought to be shared somehow so we can import it as a dependency rather than inline here